### ISSUE TITLE
Style : Social Media Icon 

### ISSUE THREAD

**Issue by AnuragRoshan** [elixir-cloud-aai/landing-page#121]
Some Social Media Icons of contributors are not aligned correctly along with that we can provide brand color on hover effect to icon for better UI .
<img width="629" alt="Screenshot 2024-02-29 104502" src="https://github.com/elixir-cloud-aai/landing-page/assets/68966298/f38f14e8-a31b-482a-ab01-d54c21524230">

**Comment by uniqueg** [comment]
Thanks a lot @AnuragRoshan - could you highlight what exactly you mean? Is the distance between the icons different for different services and/or across contributors different? Or do you mean the borders around the ORCiD icon compared to all other icons? I'm not sure I fully understand what you mean. 

As for the hovering effect: Perhaps this is a separate issue? And possibly a larger one as we would probably want a similar behavior on other links (e.g., in the footer) as well... What do you think?

**Comment by AnuragRoshan** [comment]
Yep I want to make icons align and justify center , as of now it is not aligned correctly for both ORCiD and other social media link  .
For Hovering effect : Yep providing similar behavior to footer is also great idea .

Please assign me this issue , I will do it .

**Comment by uniqueg** [comment]
Thanks, I did that. But please focus on the icons only. Then the hovering effect in another issue and PR :pray:

**Comment by AnuragRoshan** [comment]
Sure @uniqueg !

**Comment by psankhe28** [comment]
Hi @AnuragRoshan , if you are not working on this issue, can I create a PR?

**Comment by psankhe28** [comment]
@AnuragRoshan , if you are not working on this, can I work on this issue?

**Comment by Utkarsh-Anandani** [comment]
@uniqueg can you assign the issue to me?

**Comment by anuragxxd** [comment]
Hey @Utkarsh-Anandani, you are free to work on this. Free world!
